components/script/dom/element.rs

```diff
@@ -3230,6 +3230,19 @@ impl Element {
         let root = node.GetRootNode();
         root.is::<Document>()
     }
+
+    // https://html.spec.whatwg.org/multipage/#cannot-navigate
+    pub fn cannot_navigate(&self) -> bool {
+        let document = document_from_node(self);
+
+        // Step 1.
+        !document.is_fully_active() || (
+            // Step 2.
+            self.upcast::<Node>().type_id() != NodeTypeId::Element(ElementTypeId::HTMLElement(
+                HTMLElementTypeId::HTMLAnchorElement,
+            )) && !self.upcast::<Element>().is_connected()
```

Comment:
This is a method on Element, so we should be able to call `self.is_connected()`

Reply:
Ah, I forgot to update it! Thanks for catching this 😂